grails-app/controllers/org/pih/warehouse/inventory/ReplenishmentController.groovy

```diff
@@ -19,4 +21,17 @@ class ReplenishmentController {
     def create = {
         render(template: "/common/react", params: params)
     }
+
+    def print = {
+        Order transferOrder = Order.get(params.id)
+        def picklist = Picklist.findByOrder(transferOrder)
+
+        def lineItems = transferOrder.orderItems.findAll { !it.parentOrderItem }.sort { it.product.name }
+
+        def headerItems = [orderNumber: transferOrder.orderNumber,
```

Comment:
Minor nitpick ... move orderNumber to the next line and the trailing bracket to a new line. Then reformat.